src/common/constants.ts

```diff
@@ -6,6 +6,8 @@ export const slippiHomepage = "https://slippi.gg";
 export const slippiActivationUrl = "https://slippi.gg/online/enable";
 export const slippiManagePage = "https://slippi.gg/manage";
 
+export const currentRulesVersion = 1;
```

Comment:
Why does the current rules version need to be stored in the launcher? The backend knows what the latest version is, and knows the latest version the user has accepted. The only thing the frontend needs to know is whether to show the rules step (a boolean). Can't the backend just return a boolean to the frontend?

Reply:
Hmmm maybe. Just a couple things at the moment though:

1. I don't want to think too hard about it right now.
2. The one issue I can think of is that let's say we update the slippi rules and want people to accept them again, if they haven't updated their launcher and the backend says they need to see the rules again, they would see and accept an old version of the rules